src/scripts/release.zig

```diff
@@ -615,6 +616,52 @@ fn publish_node(shell: *Shell, info: VersionInfo) !void {
     });
 }
 
+fn publish_docker(shell: *Shell, info: VersionInfo) !void {
+    var section = try shell.open_section("publish docker");
+    defer section.close();
+
+    try shell.project_root.setAsCwd();
+    assert(try shell.dir_exists("dist/tigerbeetle"));
+
+    try shell.exec(
+        \\docker login --username tigerbeetle --password {password} ghcr.io
+    , .{
+        .password = try shell.env_get("GITHUB_TOKEN"),
+    });
+
+    try shell.exec(
```

Comment:
If you want to be able to test these before pushing, you can do the following:

```
# Build the Dockerfile to an OCI tarball.
docker buildx build --file tools/docker/Dockerfile . --platform linux/amd64,linux/arm64 --output type=oci,dest=dist/docker/oci.tar

# Load each platform separately, so we can test it locally.
docker import dist/docker/oci.tar --platform arm64 tigerbeetle-arm64
docker import dist/docker/oci.tar --platform amd64 tigerbeetle-amd64
docker run tigerbeetle-arm64 version --verbose
docker run tigerbeetle-amd64 version --verbose

# Build and push - this will come straight from build cache.
docker buildx build --file tools/docker/Dockerfile . --platform linux/amd64,linux/arm64 --build-arg=DEBUG=false --tag ghcr.io/tigerbeetle/tigerbeetle:{version} --push
```